### PR TITLE
Use template string to concate strings ( Refactor Part 2)

### LINKED ISSUE #21: Refactor code: Part 2
Use template literals instead of:

```
          'translate(' +
          yearScale(d.date.year()) +
          ',' +
          this.settings.tooltip_padding * 2 +
          ')'
```

etc.

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

File: src/calendar-heatmap.component.js [hunk 1]
```diff
@@ -243,13 +243,9 @@ export class CalendarHeatmap extends Component {
...
       })
       .attr('transform', (d) => {
-        return (
-          'translate(' +
-          yearScale(d.date.year()) +
-          ',' +
-          this.settings.tooltip_padding * 2 +
-          ')'
-        );
+        return `translate(${yearScale(d.date.year())},'${
```
Comment (maneetgoyal) [inline src/calendar-heatmap.component.js]:
```suggestion
        return `translate(${yearScale(d.date.year())}, ${
```

File: src/calendar-heatmap.component.js [hunk 2]
```diff
@@ -831,15 +827,9 @@ export class CalendarHeatmap extends Component {
...
-        return (
-          'translate(' +
-          weekScale(moment(d.date).week()) +
-          ',' +
-          (dayScale(moment(d.date).weekday()) +
-            dayScale.bandwidth() / 1.75 -
-            15) +
-          ')'
-        );
+        return `translate(${weekScale(moment(d.date).week())},${
```
Comment (maneetgoyal) [inline src/calendar-heatmap.component.js]:
```suggestion
        return `translate(${weekScale(moment(d.date).week())}, ${
```

File: src/calendar-heatmap.component.js [hunk 3]
```diff
@@ -1150,15 +1140,12 @@ export class CalendarHeatmap extends Component {
         return Math.min(dayScale.bandwidth(), this.settings.max_block_height);
       })
       .attr('transform', (d) => {
-        return (
-          'translate(' +
-          weekScale(moment(d.date).week()) +
-          ',' +
-          (dayScale(moment(d.date).weekday()) +
+        return `translate(${weekScale(moment(d.date).week())},
+          ${
```
Comment (maneetgoyal) [inline src/calendar-heatmap.component.js]: add a space after comma if needed

File: src/calendar-heatmap.component.js [hunk 4]
```diff
@@ -1783,13 +1770,13 @@ export class CalendarHeatmap extends Component {
     let minutes = Math.floor((seconds - hours * 3600) / 60);
     let time = '';
     if (hours > 0) {
-      time += hours === 1 ? '1 hour ' : hours + ' hours ';
+      time = hours === 1 ? `${time}1 hour ` : `${time}${hours} hours `;
     }
     if (minutes > 0) {
-      time += minutes === 1 ? '1 minute' : minutes + ' minutes';
+      time = minutes === 1 ? `${time}1 minute` : `${time}${minutes} minutes`;
```
Comment (maneetgoyal) [inline src/calendar-heatmap.component.js]: it is okay for now, but make a new issue and look into https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules/PluralRules

Comment (maneetgoyal) [inline src/calendar-heatmap.component.js]: looks incorrect, `time +=` is changed to `time =`

Comment (qur786) [reply src/calendar-heatmap.component.js]: @maneetgoyal sir I have used the time variable before both of  the conditions' output as you can see. so `time += '1 minute' is similar to `${time}1 minute`

Comment (maneetgoyal) [reply src/calendar-heatmap.component.js]: study https://github.com/manufac-analytics/reactjs-calendar-heatmap/pull/23/commits/0cc654ac2eeaa87eb278b8e30bb7fe604542eb44

**Review by maneetgoyal — APPROVED** [review]

**Pull request merged**